transcom/mymove
# B-20293: MTO Page Shipment Header Missing Values

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20293)

## Summary

The location and date information under the shipment header is being removed as requested in the feature this BL item is under.

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access office
2. Login as a TOO, TIO or counselor
3.Navigate select a move with a PPM or any other shipment
4. Navigate to the MTO page and examine a shipment
5. Notice the Shipment address information should not be there

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
Before:
![image](https://github.com/user-attachments/assets/3d877b16-090c-4a0f-af2e-f283a6734152)

After: 
![image](https://github.com/user-attachments/assets/f241ae9e-4623-40d5-941e-a9479a7f7c88)

508:
*Note the header was removed
![image](https://github.com/user-attachments/assets/00be8c65-250b-40c3-848a-346543a9f5c5)
